src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -70,8 +83,9 @@ public CheckResult Check(string expressionText, RecordType parameterType = null)
                 parameterType = new RecordType();
             }
 
-            var formula = new Formula(expressionText);
+            var formula = new Formula(expressionText, loc: Config.LanguageSettings);
 
+            // TODO: Can we enable expression chaining via some config option?
```

Comment:
Chaining is for side-effectful expressions, and we haven't yet added support for that in the interpreter. For now, Flags.None is the only supported mode

Reply:
OK. Removed.